site/src/app/home/home.component.ts

```diff
@@ -115,6 +140,18 @@ export class HomeComponent {
     this.ctx.fillStyle = fill;
   }
 
+  enableEraser() {
+    this.mode = 'eraser';
+    this.ctx.fillStyle = 'white';
+    this.ctx.strokeStyle = 'white';
+  }
+
+  enablePencil() {
+    this.mode = 'pencil';
+    this.ctx.fillStyle = 'black';
+    this.ctx.strokeStyle = 'black';
+  }
```

Comment:
Most of the changes in this PR are updating the ui but this is the actual eraser part. Sets the fill to white for the eraser and black for the pencil